packages/nestjs/src/helpers.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * Determines if the exception is an expected control flow error.
+ * - HttpException errors will have a status property
+ * - RpcException errors will have an error property
+ *
+ * @returns `true` if the exception is expected and should not be reported to Sentry, otherwise `false`.
+ */
+export function isExpectedError(exception: unknown): boolean {
```

Comment:
m: Rename to `isProbablyNestjsHttpException()`?

Reply:
I don't think that's a good name because it's already responsible for filtering both RpcExceptions and HttpExceptions and maybe others in the future so should be a more general name